File: Perl/procon.pl
#!/usr/bin/env perl

use strict;
use utf8;
use warnings;


if ($0 eq __FILE__) {
  while (defined(my $line = <STDIN>)) {
    $line =~ s/\s+$//;
    my @tokens = split(/ +/, $line);
    my $a = $tokens[0];
    my $b = $tokens[1];
    <+CURSOR+>
  }
}

File: JavaScript/Nodejs/procon.js
(function() {
  'use strict';

  var lines = [];
  require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
  }).on('line', function(line) {
    lines.push(line);
  });
  process.stdin.on('end', function() {
    lines.forEach(function(line) {
      var tokens = line.split(' ');
      console.log(tokens);
      <+CURSOR+>
    });
  });
})();
